src/worker.ts

```diff
@@ -53,14 +53,14 @@ declare var wabt: {
 
 async function loadBinaryen() {
   if (typeof Binaryen === "undefined") {
-    importScripts("../lib/binaryen.js");
+    importScripts("https://cdn.rawgit.com/AssemblyScript/binaryen.js/master/index.js");
```

Comment:
we want to fix it to specific revision

Reply:
@decode told me If you need some specific patches to original code of `binaryen` or `wabt` this not a problem.

Reply:
Instead `master` we can easily pin to specific tag. Just choose one of them: https://github.com/AssemblyScript/binaryen.js/tags

Reply:
https://cdn.rawgit.com/AssemblyScript/binaryen.js/e931e827cd586f05d25fe076e7ac71b2419f1e07/index.js ?

Reply:
Fwiw, it's also possible to do this: https://cdn.rawgit.com/AssemblyScript/binaryen.js/v46.0.0/index.js